monai/utils/type_conversion.py

```diff
@@ -239,8 +239,8 @@ def convert_data_type(
 
```

Comment:
Maybe line238 need to modify to 'Convert to `MetaTensor` /`torch.Tensor`/`np.ndarray` from `torch.Tensor`/`np.ndarray`/`float`/`int` etc.'

Reply:
thanks, this is updated.